<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmGaugeProgress</class>
 <widget class="QWidget" name="frmGaugeProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="GaugeProgress" name="gaugeProgress1" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="GaugeProgress" name="gaugeProgress2" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="GaugeProgress" name="gaugeProgress3" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="GaugeProgress" name="gaugeProgress4" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="GaugeProgress" name="gaugeProgress5" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="GaugeProgress" name="gaugeProgress6" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="ckShowPointer">
     <property name="text">
      <string>显示指针</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="ckShowValue">
     <property name="text">
      <string>显示值</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="cboxPointerStyle">
     <item>
      <property name="text">
       <string>圆形指示器</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>指针指示器</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>圆角指针指示器</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>三角指示器</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSlider" name="horizontalSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GaugeProgress</class>
   <extends>QWidget</extends>
   <header>gaugeprogress.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
